sync.py

```diff
@@ -0,0 +1,166 @@
+
+
+
+#returns files in present in a folder
+def listFiles(service, folderID):
+    results = service.files().list(
+        q="'" + folderID + "' in parents and mimeType != 'application/vnd.google-apps.folder' ",
+        fields="nextPageToken, files(id, name, size, mimeType)",
+    ).execute()
+    return results.get('files', [])
+
+
+#get course ID from the table resources_course
+def getCourseID(course):
+    try:
+        qry = '''
+                SELECT * FROM resources_course WHERE code = %s 
+                '''
+        cursor.execute(qry,(course,))
+        courseid = cursor.fetchone()
+        if courseid is None:
+            return(0)
+        else: 
+            return courseid[0]
+    except (Exception, psycopg2.Error) as error:
+        print("Failed to find course ID...", error)
+                
+
+#check whether file already exists in database
+def checkIfFileInDB(driveid):
+    try:
+        qry = '''
+            SELECT EXISTS (
+                SELECT 1
+                FROM resources_file
+                WHERE driveid = %s
+            )'''
+        cursor.execute(qry, (driveid,))
+        return cursor.fetchone()[0]
+    except (Exception, psycopg2.Error) as error:
+        print("error: ", error)
+        return False
+
+
+#add new file to the database
+def addFileInDB(file):
+    try:
+        qry = '''
+                INSERT INTO resources_file (title, driveid, downloads, size, date_modified, fileext, filetype, finalized, course_id) VALUES (%s,%s,%s,%s,%s,%s,%s,%s,%s)
+            '''
+        recordToInsert = (file['title'], file['driveid'], 0, file['size'], file['dateModified'], file['fileext'], file['filetype'], 't', file['courseid'] )
+        cursor.execute(qry, recordToInsert)
+        connection.commit()
+        print(file['title'] + ' added in database...')
+    except (Exception, psycopg2.Error) as error:
+        print("Failed to insert record", file['title'], file['courseid'], error)
+    
+
+def printContents():
+    try:
+        qry = '''
+            SELECT * FROM resources_file 
+            '''
+        cursor.execute(qry)
+        records = cursor.fetchall()
+
+        for row in records:
+            print (row[0],"\t",row[1],"\t",row[2],"\t",row[3],"\t",row[4],"\t",row[5],"\t",row[6],"\t",row[7],"\t",row[8],"\t",row[9])
+
+    except (Exception, psycopg2.Error) as error:
+        print("Failed to print", error)
+    
+
+def syncAction(file):
+    global filesSynced
+    fileExists = checkIfFileInDB(file['driveid'])
+    if(fileExists):
+        print(file['title'] + ' file already in database...')
+            
+    else:
+        addFileInDB(file)
+        filesSynced = filesSynced + 1
+        
+            
+def courseScan(course, departments, depts):
+    service = driveinit()
+    for folder in departments[depts][course]:
+        if course != 'id':
+            if folder != 'id':
+                pickFolder = departments[depts][course][folder]
```

Comment:
I don't see us separating the `_review` folders out, we should ignore the files existing in those folders